frontend/src/styles.scss

```diff
@@ -1,2 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
+@import "../node_modules/@angular/material/prebuilt-themes/deeppurple-amber.css";
```

Comment:
jeżeli import to z konwencją "/~@angular/...", ale linijka niżej i tak go nadpisze, więc nie wiem czy ma on sens

Reply:
ok usunięte